src/components/aeRoundedContainer/index.js

```diff
@@ -0,0 +1,5 @@
+import aeIdentityAvatar from './aeRoundedContainer.vue'
```

Comment:
`aeIdentityAvatar` should be replaced with `aeRoundedContainer`

Reply:
Whoops show did that slip in :(